fe/lib/components/JobCategorySuggest/JobCategorySuggestChoices.tsx

```diff
@@ -73,7 +78,7 @@ const JobCategorySuggestChoices = forwardRef<HTMLInputElement, Props>(
         <RadioGroup
           {...restProps}
           id="job-category-suggest-select"
-          name={name}
+          name={selectedJobCategory === 'Other' ? '' : name}
```

Comment:
Hmm I'm not getting the point of this change, can you explain?

Reply:
Sorry I didn't realise there were comments! I think the reason we had to do that was because I wasn't sure which value would be taken from which HTML element on form submit 😅  i.e. if I let both the radiogroup and select components have the `name`. (I'm assuming that's how `react-hook-forms` is pulling out input value?)

Reply:
ohh I see, happy to give this a shot then

Reply:
Hmm actually after delving deeper into the `react-hook-form` docs I may be wrong, I'll verify this lol

Reply:
So the good news is that I'm pretty sure we don't need that, as we're treating these as controlled components. However I have noticed that the component is in a bit of weird state where I'm unsure of whether we wanted to allow it to be both uncontrolled/controlled. 

Right now `JobCategorySuggest` won't actually work if you don't pass it a `onSelect` prop (despite it being optional), so I'm thinking of making that a required prop and just asserting that `JobCategorySuggest` should be a controlled component. The main reasons are that the underlying `RadioGroup` component expects to be controlled as well as this new feature won't work without it being controlled.

Reply:
Sounds good!